drivers/wifi/nrf700x/zephyr/src/zephyr_net_if.c

```diff
@@ -68,6 +68,15 @@ enum wifi_nrf_status wifi_nrf_if_carr_state_chg(void *os_vif_ctx,
 
 	vif_ctx_zep->if_carr_state = carr_state;
 
+	if (vif_ctx_zep->zep_net_if_ctx) {
+		if (carr_state == WIFI_NRF_FMAC_IF_CARR_STATE_ON) {
```

Comment:
In terms of Wi-Fi, what `WIFI_NRF_FMAC_IF_CARR_STATE_ON` represnts, i. e. what is considered as carrier on state? Is the interface `dormant` state handled somehow?

Reply:
This indicates connection complete including any security handshakes.